pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -1815,14 +1850,8 @@ func createMarkAndAcceptChain(aclArgs types.AppNetworkACLArgs,
 	rule2 := []string{"-A", name, "-t", "mangle", "-m", "mark", "!", "--mark", "0",
 		"-j", "ACCEPT"}
 
-	rule3 := []string{}
-	if marking == -1 {
-		rule3 = []string{"-A", name, "-t", "mangle", "-j", "CONNMARK", "--set-mark",
-			"0xffffffff"}
-	} else {
-		rule3 = []string{"-A", name, "-t", "mangle", "-j", "CONNMARK", "--set-mark",
-			strconv.FormatInt(int64(marking), 10)}
-	}
+	rule3 := []string{"-A", name, "-t", "mangle", "-j", "CONNMARK", "--set-mark",
+		strconv.FormatInt(int64(marking), 10)}
```

Comment:
Is -1 and maxint32 handled identically by connmark?

Reply:
good catch, I forgot to change it to `FormatUint`.

Reply:
done